config/crd/bases/policies.kubewarden.io_admissionpolicies.yaml

```diff
@@ -1,9 +1,10 @@
+
 ---
 apiVersion: apiextensions.k8s.io/v1
 kind: CustomResourceDefinition
 metadata:
   annotations:
-    controller-gen.kubebuilder.io/version: v0.8.0
+    controller-gen.kubebuilder.io/version: v0.4.1
```

Comment:
It looks like you used an older version of kubebuilder... This gives me doubts about the code that has been automatically generated. Am I a fool?

Reply:
This commit is not PR branch. You need to rebase it or cherry pick it.

Reply:
![image](https://user-images.githubusercontent.com/1514798/172607934-c3a72c55-ffa0-495e-ab06-0c255c5e1886.png)